tests/driver.sh

```diff
@@ -533,4 +533,18 @@ int main()
 }
 EOF
 
+try_ 1 << EOF
+#if -1
+#define A 1
```

Comment:
My bad, there should be no same value for different branch. Furthermore, returning 1 is not ideal since compilation error will also returns 1 as result, will fix this in later commit.

The patch will be committed in #111.